pkg/rpc/server/server.go

```diff
@@ -1381,7 +1379,7 @@ func (s *Server) getStateRoot(ps request.Params) (interface{}, *response.Error)
 		}
 	}
 	if err != nil {
-		return nil, response.NewRPCError("Unknown state root.", "", err)
+		return nil, response.NewRPCError("Unknown state root", err)
```

Comment:
Does the `err` here make any sense for the user?

Reply:
Well, I still think that we'd better keep it here, because it can contain not only `Key not found`, but also uint256 unmarshalling error as far as stateroot decoding error (which is hardly possible, but still may occur).

Dopped for now.